guides/v2.1/get-started/order-tutorial/order-create-quote.md

```diff
@@ -34,7 +34,7 @@ Magento identifies three types of users that can create a shopping cart:
 * A guest user. These users could be customers who haven't logged in yet, or they could be users who have no intention of creating an account. An anonymous user's cart is called a guest cart.
 
 ### Create a cart for a logged-in customer {#create-cart}
-This tutorial manages a the cart of a logged-in customer. Unless otherwise noted, all calls must specify customer's token `q0u66k8h42yaevtchv09uyy3y9gaj2ap` in the header.
+This tutorial manages a cart of the logged-in customer. Unless otherwise noted, all calls must specify customer's token `q0u66k8h42yaevtchv09uyy3y9gaj2ap` in the header.
```

Comment:
The sentence should be "This tutorial manages the cart of a logged-in customer."

Reply:
@keharper I've changed the sentence.